x-pack/plugins/data_usage/server/routes/internal/usage_metrics_handler.ts

```diff
@@ -34,44 +36,27 @@ export const getUsageMetricsHandler = (
       const core = await context.core;
       const esClient = core.elasticsearch.client.asCurrentUser;
 
-      // @ts-ignore
-      const { from, to, metricTypes, dataStreams: dsNames, size } = request.query;
+      const { from, to, metricTypes, dataStreams: requestDsNames } = request.query;
       logger.debug(`Retrieving usage metrics`);
 
       const { data_streams: dataStreamsResponse }: IndicesGetDataStreamResponse =
         await esClient.indices.getDataStream({
-          name: '*',
+          name: requestDsNames,
           expand_wildcards: 'all',
         });
```

Comment:
We won't need a call to `getDataStream` now that we're going to have dataStreams in the req. That's the list we should use to request autoOps API.

Reply:
Do you remember our chat on Mon where we discussed needing to make sure a user who uses this API has permission to access the data streams its requesting?

Reply:
Oh yes, that makes sense. But we can also just verify and provide only an allowable list of datastreams on the UX and the datastreams API that provides that. That way users won't be confused why some of the datastreams they requested were not included.  What do you think?